scss/style.scss

```diff
@@ -0,0 +1,1241 @@
+body {
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+}
+
+.wpmdb {
+  .wrapper {
+    max-width: 1280px;
+    margin: 100px;
+  }
```

Comment:
This seems to be just for displaying the component library and not for use in the plugin styles? Should probably be part of `component-layout.scss`.

Reply:
How do you mean sorry? Are you saying this won't be used and is just for display purposes? So should be moved to component-layout.scss?

Reply:
Yes